server/secret_server.go

```diff
@@ -60,21 +60,21 @@ func (s *SecretServer) GetSecret(ctx context.Context, req *api.GetSecretRequest)
 	return apiSecret(secret), nil
 }
 
-func (s *SecretServer) ListSecrets(ctx context.Context, req *api.ListSecretsRequest) (secrets *api.ListSecretsResponse, err error) {
-	var modelSecrets []*model.Secret
-	modelSecrets, err = s.resourceManager.ListSecrets(req.Namespace)
-	if err != nil {
-		return nil, util.NewUserError(codes.Unknown, "Error encountered with listing secrets.")
+func (s *SecretServer) ListSecrets(ctx context.Context, req *api.ListSecretsRequest) (*api.ListSecretsResponse, error) {
```

Comment:
Removed named return values so you don't have to come up with new variable names (optional)